### ISSUE TITLE
Nominations: `!talentpool queue` command

### ISSUE THREAD

**Issue by wookie184** [python-discord/bot#2303]
This would allow queueing a user to be reviewed next rather than using the autoreviewer. A mod could run `!tp q 493839819168808962` and that user would be reviewed when the reviewer is next ready, returning to the autoreviewer after the queue is empty.

We should also have a `!tp queue clear` command to clear the queue, and a `!tp queue list` command to show the current queue members.

**Comment by wookie184** [comment]
Thoughts on what the best way to persist this queue would be?

**Comment by shtlrs** [comment]
> Thoughts on what the best way to persist this queue would be?

Can't we use a nomination queue that would be a bot variable ?

This would of course disappear once the bot dies, which I don't think is that impactful, but avoids the complexity of having an external tool.

**Comment by wookie184** [comment]
Redis could work, although it's not ideal to rely on it for persistence. It might be fine for this though, unless there's a nice way we can store it in the site database, not entirely sure how would be best to represent a queue in the database though.

**Comment by shtlrs** [comment]
I've made a quick research on how this could be done in Postgres, and I've found quite a good [example](https://blabosoft.com/implementing-queue-in-postgresql).

Implementation would only depend on how resilient & precise we want it to be (in terms of consistency & persistence)
Using Redis leverages the fact that we don't need to set up this model & endpoints in our site project if we'd go with having a queue in our DB, but also the fact that it's simpler to implement but would die whenever our Redis instance is restarted/redeployed.

**Comment by wookie184** [comment]
@vivekashok1221 Yeah I think Postgres is probably the way to go, I've assigned you 👍

**Comment by vivekashok1221** [comment]
Thank you Wookie.
Just letting you know, I'll need a few days to address this issue as I have some stuff going on right now.

**Comment by yung613** [comment]
It's a good one
